use_cases/eluc/predictors/sklearn/sklearn_predictor.py

```diff
@@ -41,21 +40,22 @@ def save(self, path: str):
         else:
             save_path = path
         save_path.mkdir(parents=True, exist_ok=True)
-        config = {
-            "features": self.features,
-            "label": self.label
-        }
         with open(save_path / "config.json", "w", encoding="utf-8") as file:
-            json.dump(config, file)
+            json.dump(self.config, file)
```

Comment:
We now dump all the arguments we used to create the model instead of just the ones we use at inference time